extensions/wikia/TemplateClassification/TemplateClassification.hooks.php

```diff
@@ -91,17 +91,13 @@ public function onEditPageMakeGlobalVariablesScript( array &$aVars ) {
 	public function onEditPageShowEditFormFields( \EditPage $editPage, \OutputPage $out ) {
 		global $wgCityId;
 
-		if ( $out->getSkin() instanceof \SkinMonoBook ) {
-			return true;
-		}
+		$context = \RequestContext::getMain();
 
-		$templateType = $this->getTemplateTypeForEdit( $editPage->getTitle(), $wgCityId );
+		if ( ( new Permissions() )->shouldDisplayEntryPoint( $context->getUser(), $context->getTitle() ) ) {
+			$templateType = $this->getTemplateTypeForEdit( $editPage->getTitle(), $wgCityId );
 
-		$editPage->addHiddenField([
-			'name' => 'templateClassificationType',
-			'value' => $templateType,
-			'type' => 'hidden',
-		]);
+			$out->addHTML( \Html::hidden( 'templateClassificationType', $templateType ) );
```

Comment:
Yes, this hook is run while form is created. See this function https://github.com/Wikia/app/blob/dev/includes/EditPage.php#L1764